deepchem/models/torch_models/atomic_conv.py

```diff
@@ -0,0 +1,238 @@
+import torch
+import numpy as np
+from deepchem.models.torch_models.torch_model import TorchModel
+from deepchem.models.torch_models.layers import AtomicConvolutionModule
+from deepchem.models.losses import L2Loss
+from deepchem.metrics import to_one_hot
+
+from typing import Callable, Optional, Sequence
+from deepchem.utils.typing import OneOrMany, ActivationFn
+
+
+class AtomConv(TorchModel):
```

Comment:
You should name this the same as the existing TF model and match all arguments (we will do a swap and replace in a future version)